jsonpointer.py

```diff
@@ -32,38 +32,22 @@
 
 """ Identify specific nodes in a JSON document (RFC 6901) """
 
-from __future__ import unicode_literals
-
 # Will be parsed by setup.py to determine package metadata
 __author__ = 'Stefan Kögl <stefan@skoegl.net>'
-__version__ = '2.4'
+__version__ = '3.0.0'
```

Comment:
Any specific reason to go for 3 digits? Is there some standard to follow?

Reply:
[semver](https://semver.org/)